content/docs/integrations/aws-resources/index.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Configuring AWS Resources Autodiscovery
+publishedDate: '2022-03-29T21:00:00.0Z'
+description: How to configure Roadie to read AWS resources
+
+humanName: AWS Resources
+logoImage: '../../../assets/logos/aws/Amazon_Web_Services_Logo.png'
+integrationType: Integration
+---
+
+## Introduction
+
+Roadie has the capability to automatically ingest resources from AWS. This is done by automatic discovery and currently ingest all configured resource types per AWS region.
+
+This guide describes how to set up Roadie to access your specific AWS resources and automatically ingest them.
+
+
+##  Step 1: Get the roadie IAM details
+
+Navigate to `Administration > Settings > AWS Resources` and make a note of the Roadie backend role ARN and account ID. This is mentioned on the title text of the AWS Resources integration settings page.
+
+##  Step 2: Create a federated role in your account for Roadie
+
+Follow the steps [here](/docs/details/accessing-aws-resources) to create the role. 
+
+The role needs to follow this naming convention `arn:aws:iam::*:role/<your-tenant-name>-roadie-read-only-role` where <your-tenant-name> matches your organisation's name used in the url of your Roadie instance.
+
+You'll need to attach policies to the role to be able to retrieve information about the resources you want ingested. The supported resources and their needed policies are listed in a table at the end of this page. You can use the same role for multiple resource types as long as the needed permissions are granted to it.
+
+For quick experimentation, you can use `AWS<ResourceType>ReadOnlyAccess` policies provided by AWS, but the best practice is to allow only specific needed operations.
+
+
+
+
+##  Step 3: Configure your Roadie instance to use the new role
+
+![AWS Resources config](aws-resources-config.png)
+
+On the AWS Resources settings page `Administration > Settings > Aws Resources` in Roadie click `Add Item`. 
+Here you can select the type of resource you want to be automatically ingested. The possible options are listed in the table at the bottom of the page.
```

Comment:
The word automatically is repeated so many times. Possibly it could be dropped from this paragraph?